libctru/include/3ds/services/am.h

```diff
@@ -25,41 +58,84 @@ void amExit(void);
 Handle *amGetSessionHandle(void);
 
 /**
- * @brief Gets the number of titles for a given mediatype.
- * @param mediatype Mediatype to get titles from.
+ * @brief Gets the number of titles for a given media type.
+ * @param mediatype Media type to get titles from.
  * @param count Pointer to write the title count to.
  */
-Result AM_GetTitleCount(u8 mediatype, u32 *count);
+Result AM_GetTitleCount(u32 *count, FS_MediaType mediatype);
```

Comment:
Seemed like a good idea to standardize the functions on outputs followed by inputs. I can revert these kinds of changes if you'd like.